docs/usage-guide/optional-questions-and-branching.md

```diff
@@ -16,7 +19,8 @@ adventure. These choices introduce diverging paths of interactions that may or m
 ### Optional Questions
 
 The following is a basic example that has two optional questions that are not asked based on the answer to the first
-question.
+question. It also has an optional "echo" that is only displayed
```

Comment:
I wasn't referring to a class by name just as "two optional questions" wasn't referring to `BasicQuestion`s. I used quotation marks because it is otherwise awkward to use _echo_ as a noun here. I'll leave it as-is unless you have strong feelings (which it seems you don't).

Reply:
Hmm. Maybe something "It also has an optional message that is only...". That phrasing could remove the confusion I had.